src/webui/www/private/addwebseeds.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="${LANG}">
+
+<head>
+    <meta charset="UTF-8" />
+    <title>QBT_TR(Add web seeds)QBT_TR[CONTEXT=HttpServer]</title>
+    <link rel="stylesheet" href="css/style.css?v=${CACHEID}" type="text/css" />
+    <script src="scripts/lib/MooTools-Core-1.6.0-compat-compressed.js"></script>
+    <script src="scripts/lib/MooTools-More-1.6.0-compat-compressed.js"></script>
+    <script>
+        "use strict";
+
+        window.addEvent("domready", () => {
+            new Keyboard({
+                defaultEventType: "keydown",
+                events: {
+                    "Escape": function(event) {
+                        window.parent.qBittorrent.Client.closeWindows();
+                        event.preventDefault();
+                    },
+                    "Esc": function(event) {
+                        window.parent.qBittorrent.Client.closeWindows();
+                        event.preventDefault();
+                    }
+                }
+            }).activate();
+
+            $("urls").focus();
+            $("addWebseedsButton").addEvent("click", (e) => {
+                new Event(e).stop();
+                const hash = new URI().getData("hash");
+                new Request({
+                    url: "api/v2/torrents/addWebseeds",
+                    method: "post",
+                    data: {
+                        hash: hash,
+                        urls: $("urls").value.split("\n").map(w => encodeURIComponent(w.trim())).filter(w => (w.length > 0)).join("|")
+                    },
+                    onComplete: function() {
+                        window.parent.qBittorrent.Client.closeWindows();
+                    }
+                }).send();
+            });
+        });
+    </script>
+</head>
+
+<body>
+    <div style="text-align: center;">
+        <br />
+        <h2 class="vcenter">QBT_TR(List of web seeds to add (one per line):)QBT_TR[CONTEXT=HttpServer]</h2>
```

Comment:
Since PR #21097 you'll need to provide a label for each textarea/input element.
```suggestion
        <label for="urls">QBT_TR(List of web seeds to add (one per line):)QBT_TR[CONTEXT=HttpServer]</label>
```